cmd/kube-controller-manager/app/options/options_test.go

```diff
@@ -170,8 +168,8 @@ func TestAddFlags(t *testing.T) {
 	expected := &KubeControllerManagerOptions{
 		Generic: &cmoptions.GenericControllerManagerConfigurationOptions{
 			GenericControllerManagerConfiguration: &cmconfig.GenericControllerManagerConfiguration{
-				Port:            10252,     // Note: InsecureServingOptions.ApplyTo will write the flag value back into the component config
-				Address:         "0.0.0.0", // Note: InsecureServingOptions.ApplyTo will write the flag value back into the component config
+				Port:            10257,     // Note: SecureServingOptions.ApplyTo will write the flag value back into the component config
+				Address:         "0.0.0.0", // Note: SecureServingOptions.ApplyTo will write the flag value back into the component config
```

Comment:
I expected these to be pruned everywhere. They are old i.e. insecure only, aren't they?

Reply:
okay I will remove these.